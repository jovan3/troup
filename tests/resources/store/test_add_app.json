{"test-app": {"name": "test-app", "command": "run test-app", "description": "description", "params": {"p1": "v1", "p2": "v2"}, "needs": {"memory": 1, "cpu": 0, "disk": 3, "network": 2}}}