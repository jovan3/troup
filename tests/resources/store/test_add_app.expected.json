{
    "test-app": {
        "name": "test-app",
        "description": "description",
        "command": "run test-app",
        "params": {
            "p1": "v1",
            "p2": "v2"
        },
        "needs": {
            "cpu": 0,
            "memory": 1,
            "network": 2,
            "disk": 3
        }
    }
}
